swmmio/core.py

```diff
@@ -751,6 +755,23 @@ def evaporation(self, df):
         """Set inp.evaporation DataFrame."""
         self._evaporation_df = df
 
+    @property
+    def losses(self):
```

Comment:
Pretty sure we could abstract away this repetitive logic - at least for the most trivial cases